specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -0,0 +1,861 @@
+        "x-ms-examples": {
+          "GetLedgerEntries": {
+            "$ref": "./examples/ledger/GetLedgerEntries.json"
+          }
+        },
+        "x-ms-pageable": {
+          "itemName": "entries",
+          "nextLinkName": "@nextLink"
+        },
+        "summary": "Gets ledger entries from a sub-ledger corresponding to a range.",
+        "description": "A sub-ledger id may optionally be specified. Only entries in the specified (or default) sub-ledger will be returned.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubLedgerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TransactionIdFromParameter"
+          },
+          {
+            "$ref": "#/parameters/TransactionIdToParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ledger entries in the specified range.",
+            "schema": {
+              "$ref": "#/definitions/PagedLedgerEntries"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "PostLedgerEntry",
+        "x-ms-examples": {
+          "PostLedgerEntry": {
+            "$ref": "./examples/ledger/PostLedgerEntry.json"
+          }
+        },
+        "summary": "Writes a ledger entry.",
+        "description": "A sub-ledger id may optionally be specified.",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubLedgerIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "entry",
+            "description": "Ledger entry.",
+            "schema": {
+              "$ref": "#/definitions/LedgerEntry"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the transaction id at which the write will be durably stored.",
+            "headers": {
+              "x-ms-ccf-transaction-id": {
+                "type": "string",
+                "description": "The transaction id at which this write will become durable."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/LedgerWriteResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "common.json#/definitions/ConfidentialLedgerError"
+            }
+          }
+        }
+      }
+    },
+    "/app/transactions/{transactionId}": {
+      "get": {
+        "operationId": "GetLedgerEntryForTransactionId",
```

Comment:
Since there are no other `GetLedgerEntryBy*` operations, could we rename this to `GetLedgerEntry`?

Reply:
Done!